<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CW</class>
 <widget class="QWidget" name="CW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollBar" name="horizontalScrollBar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>530</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_DispGrid">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>显示网格</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_DispDataPoint">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>570</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>突出显示数据点</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_DispCoor">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>570</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>显示点的坐标</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_XaxisLen">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>570</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_YaxisLen">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>570</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>570</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X轴显示点数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>570</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y轴显示点数</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Ydiv">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>570</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>570</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y轴分隔数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>570</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X轴分隔数</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Xdiv">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>570</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="Waveview" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>1000</width>
     <height>480</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>20</y>
     <width>25</width>
     <height>480</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Reset">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>570</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_AA">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>570</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>抗锯齿</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Waveview</class>
   <extends>QWidget</extends>
   <header location="global">waveview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
